app/templates/macros/nav_macros.html

```diff
@@ -17,19 +17,36 @@
     {% endif %}
 {% endmacro %}
 
-{% macro account_items(current_user) %}
+{% macro action_items(current_user) %}
```

Comment:
can we think of a better name for this (feel free to say no)? maybe like `page_items` or something to do with pages.